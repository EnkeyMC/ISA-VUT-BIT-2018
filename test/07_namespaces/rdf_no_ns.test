NAME=RDF without specified namespace
ARGS=http://www.mocky.io/v2/5beefa592e00006435eeeb5c
RC=0
CHECK_OUTPUT=1
