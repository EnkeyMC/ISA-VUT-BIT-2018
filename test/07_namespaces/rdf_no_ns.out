*** XML.com ***
Processing Inclusions with XSLT
Putting RDF to Work
